versioned_docs/version-1.3.0/install-kubeslice/kubeslice-cli/topology-configuration.mdx

```diff
@@ -50,8 +50,9 @@ The following table describes the helm chart configuration parameters.
 
 | Parameter                                                                 | Parameter Type | Description                                                                                    | Required  |
 | ------------------------------------------------------------------------- | -------------- | ---------------------------------------------------------------------------------------------- | --------- |
-| repo_alias                                                                | String         | The alias of the helm repo for KubeSlice charts.                                               | Optional  |
-| repo_url                                                                  | String         | The URL of the helm charts for KubeSlice.                                                      | Mandatory |
+| repo_alias                                                                | String         | The alias of the helm repo for KubeSlice charts. To use local chart provide the path to the charts.  | Optional  |
+| repo_url                                                                  | String         | The URL of the helm charts for KubeSlice.                                                      | Optional |
+| use_local                                                                 | Boolean        | This is to use local helm chart in the Air-Gapped environment. Set the value to true to use local charts. The default value is false. | Optional|
```

Comment:
This is to use the local helm chart ...

Set the value to `true` Use backticks (```) for both true and false